### DISCUSSION TITLE
skip checkup passes, but I'm unable to compile HelloSkip

### DISCUSSION THREAD

**Discussion by cromotron** [skiptools/skip#98] [Q&A]
I receive the following error message 
An error occurred while performing transpilation: expected colon, near ""

**Comment by marcprux** [comment]
Does this error happen in Xcode or on the console? Can you post a screenshot? Also, can you post the output of `skip checkup`, as well as the results of `skip export` when run in your project's folder?

This should give us enough context to start investigating.

**Reply by dgnsstyles** [reply]
Many versions and doesn't not work with skip export in the hello-skip and skip checkup upgrading the versions as well.

**Reply by marcprux** [reply]
Can you post the output?

**Comment by marcprux** [comment]
Better late than never: as discussed in https://github.com/skiptools/skip/issues/111, I think the problem may be that your `~/.skiptools/skipkey.env` file isn't valid YAML. If you have a key in there, can you make sure it is in the form:

```yaml
SKIPKEY: SKP1234
```
